frontend/src/components/common/TooltipIcon.scss

```diff
@@ -1,12 +1,25 @@
 @import '../../colors.scss';
+
 .tooltip-icon {
-  color: $icon-light-color;
+  min-width: 14px;
+  min-height: 14px;
+  margin-left: -14px;
+  color: $secondary-variant-color;
 }
 
 .tooltip {
   white-space: pre-wrap;
   .tooltip-inner {
     text-align: left;
     max-width: 25rem;
+    background-color: $secondary-variant-color;
+    color: #ffffff;
+  }
+  .arrow:before {
+    border-top-color: $secondary-variant-color;
```

Comment:
Can you confirm that this works if the tooltip is drawn to the left, right or bottom of the icon?

Reply:
it seems it doesn't, I'll fix that

Reply:
it doesn't, but from what I've seen all of our tooltips are from the top (the only one on the right is being removed)

Reply:
@agalbrai